packages/diagrams/frontend/sirius-components-diagrams/src/renderer/node/FreeFormNode.tsx

```diff
@@ -133,6 +171,27 @@ export const FreeFormNode: NodeComponentsMap['freeFormNode'] = memo(
       return labelId;
     };
 
+    let actionsSection: JSX.Element | null = null;
+    if (data.isHovered && actions) {
+      actionsSection = (
+        <>
```

Comment:
This fragment is not useful, it has only one child element

Reply:
done